### ISSUE TITLE
Invoking `useObservables` causes React to warn me about not using UNSAFE_componentWillReceiveProps in strict mode.

### ISSUE THREAD

**Issue by dsemelianov** [Nozbe/WatermelonDB#1442]
I'm integrating WatermelonDB into my React project. Everything is fine until I add my first invocation of `useObservables`, like so:

> const enhance = withObservables([], () => ({
>   senders: db.collections.get<Sender>("senders").query().observe(),
> }));
> 
> export const EnhancedUnsubscribeTab = enhance(UnsubscribeTab);

Whenever I run the project and get to the point where the `EnhancedUnsubscribeTab` component is rendered, React throws this error in the console:

<img width="653" alt="Screen Shot 2022-11-27 at 1 07 09 AM" src="https://user-images.githubusercontent.com/8390267/204113614-ed600a86-2e2b-4bf1-880d-d78bd725f98c.png">

Is this normal?

**Comment by radex** [comment]
Yes it is. @dsemelianov Can you add to docs-website information that it _is_ normal?
